### PR TITLE
Surface Javascript sample event

### PR DESCRIPTION
**Pull request by ehfeng**
Surfacing our js sample event for new projects. Hack until on-boarding improves (should be able to trigger multiple language samples in the future)

/cc @macqueen 

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/3715)

### CODE REVIEW

File: src/sentry/static/sentry/app/views/stream.jsx [hunk 1]
```diff
@@ -501,6 +501,7 @@ const Stream = React.createClass({
   renderAwaitingEvents() {
     let org = this.getOrganization();
     let project = this.getProject();
+    let sampleIssueId = this.state.groupIds[0];
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/stream.jsx]: this will break if theres no actual issues recorded

**Comment by ehfeng** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 504 \[r1\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMaGUL3FULszJILuxwu-r1-504:-KMaMuEpMJFJhriI6IYc:-238307076) ([raw file](https://github.com/getsentry/sentry/blob/7ce6729ff70f67d77cdc3e726cda36e2307d0fd6/src/sentry/static/sentry/app/views/stream.jsx#L504)):*
<details><summary><i>Previously, dcramer (David Cramer) wrote…</i></summary>
> this will break if theres no actual issues recorded
</details>
Creating a sample issue is part of project creation flow.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by ehfeng** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 504 \[r1\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMaGUL3FULszJILuxwu-r1-504:-KMaN4BfRoQAAc9bHCQK:-1852417572) ([raw file](https://github.com/getsentry/sentry/blob/7ce6729ff70f67d77cdc3e726cda36e2307d0fd6/src/sentry/static/sentry/app/views/stream.jsx#L504)):*
<details><summary><i>Previously, ehfeng (Eric Feng) wrote…</i></summary>
> Creating a sample issue is part of project creation flow.
</details>
I can fix it all the same.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by ehfeng** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 504 \[r1\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMaGUL3FULszJILuxwu-r1-504:-KMaO8XJsrA6LFcgr5kf:-499146009) ([raw file](https://github.com/getsentry/sentry/blob/7ce6729ff70f67d77cdc3e726cda36e2307d0fd6/src/sentry/static/sentry/app/views/stream.jsx#L504)):*
<details><summary><i>Previously, ehfeng (Eric Feng) wrote…</i></summary>
> I can fix it all the same.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by benvinegar** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 506 \[r2\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMegcaAzAnUPAtXk2wG:-KMegcaBfWUPhYwkVJzF:1581952718) ([raw file](https://github.com/getsentry/sentry/blob/c01b6b0ce37c3470e314f804fce01f089819f5f8/src/sentry/static/sentry/app/views/stream.jsx#L506)):*
> ```JSX
>     let sample_link = null;
> 
>     if (this.state.groupIds[0]) {
> ```

Just to be super safe I'd do:

```javascript
if (this.state.groupIds.length > 0) {
```

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by ehfeng** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 506 \[r2\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMegcaAzAnUPAtXk2wG:-KMegvuQWm6KSs-Br64g:676635175) ([raw file](https://github.com/getsentry/sentry/blob/c01b6b0ce37c3470e314f804fce01f089819f5f8/src/sentry/static/sentry/app/views/stream.jsx#L506)):*
<details><summary><i>Previously, benvinegar (Ben Vinegar) wrote…</i></summary>
> Just to be super safe I'd do:
> 
> ```javascript
> if (this.state.groupIds.length > 0) {
> ```
</details>
Done

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by benvinegar** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 504 \[r3\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMei7LEyA45twKstatV:-KMei7LFHD_ri6Yzv5MK:1642581071) ([raw file](https://github.com/getsentry/sentry/blob/db3c18c9bf410ceade0d7040a747134f7d76f08a/src/sentry/static/sentry/app/views/stream.jsx#L504)):*
> ```JSX
>     let org = this.getOrganization();
>     let project = this.getProject();
>     let sample_link = null;
> ```

We use `camelCase` for variable names in JavaScript (hmm, why isn't this a lint rule). Please change to `sampleLink`?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by ehfeng** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/sentry/static/sentry/app/views/stream.jsx, line 504 \[r3\]](https://reviewable.io:443/reviews/getsentry/sentry/3715#-KMei7LEyA45twKstatV:-KMfZrffHiURNAXk8x5w:-499146009) ([raw file](https://github.com/getsentry/sentry/blob/db3c18c9bf410ceade0d7040a747134f7d76f08a/src/sentry/static/sentry/app/views/stream.jsx#L504)):*
<details><summary><i>Previously, benvinegar (Ben Vinegar) wrote…</i></summary>
> We use `camelCase` for variable names in JavaScript (hmm, why isn't this a lint rule). Please change to `sampleLink`?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by benvinegar** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by macqueen** [comment]
Reviewed 1 of 1 files at r4.
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Comment by macqueen** [comment]
Reviewed 1 of 1 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3715)*

**Pull request merged**